### DISCUSSION TITLE
Tree Identation

### DISCUSSION THREAD

**Discussion by st11x** [thunderclient/thunder-client-support#805] [Q&A]
Hi,

Is it possible to adjust the indentation level for the tree in this example with the 2 highlighted notes. It will help to make it more obvious (for me) that Cat Indices is not part of the group.  

![image](https://user-images.githubusercontent.com/7398499/192598627-d8c70b1c-4954-489b-810c-1cbf105d1d05.png)

thanks
  Matt

**Comment by rangav** [comment]
Thanks for the feedback @st11x 

can you please create new feature request for this, will add to roadmap 

will provide new setting for this to control indentation
